ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -2587,6 +2587,24 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-shared-to-local-gateway-mode-migration
+- as: vsphere-ipi-permission-f28
```

Comment:
I think we can merge the minimal permission testing with other existing ci jobs to save some ci jobs.

Reply:
which job do you think is good to merge? What about "vsphere-ipi-proxy-fips-f28"

Reply:
Okay for me.